man/resolved.conf.xml

```diff
@@ -264,6 +264,25 @@
         in use.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DNSStubListenerExtra=</varname></term>
+        <listitem><para>Takes a boolean argument or one of <literal>udp</literal> and <literal>tcp</literal>. If
+        <literal>udp</literal>, a DNS stub resolver will listen for UDP requests on address configured in
+        <varname>DNSStubListenerAddress=</varname>. If <literal>tcp</literal>, the stub will listen for TCP requests
+        on the same address and port. If <literal>yes</literal> (the default), the stub listens for both UDP and TCP requests.
+        If <literal>no</literal>, the stub listener is disabled</para></listitem>
+      </varlistentry>
```

Comment:
hmm, might be something to fold into DnsStrubListenerAddress= below, i.e. that you have a syntax of:

```
DNSStubListenerAddress=1.2.3.4:88
DNSStubListenerAddress=tcp:1.2.3.4:99
DNSStubListenerAddress=udp:5.6.7.8:77
```
i.e. prefix the address with the protocol, and if the prefix is missing just listen on both.

But actually I'd rename DNSStrubListenerAddress= to DNSStubListenerExtra= then